doc/python-type-hinting.md

```diff
@@ -80,8 +78,41 @@ This import is not necessary in Python 3.10.
 
 https://www.python.org/dev/peps/pep-0563/
 
+## Type Ignore
+
+To mark portions of the program that should not be covered by type hinting, use the following on a particular line:
+
+```python
+# type: ignore
+```
+
+A `# type: ignore` comment on a line by itself at the top of a file silences all errors in the file.
+
+`# type: ignore` should only be used when unavoidable. Ensure that a comment is added to explain why it has been used.
```

Comment:
May want to specifically state that the comment used to explain the type ignore needs a prefix of `Type ignore:` so that we can tell what the comment is for and allow us to easily find it future.
For example:
"Ensure that a comment is added to explain why it has been used and have a prefix of `Type ignore:`"

Reply:
Agreed.

Reply:
I don't think this has been done?

Reply:
I have now updated this section as suggested above